### ISSUE TITLE
version 1.5 is installed. Upgrade to version 1.42.

### ISSUE THREAD

**Issue by magnolia61** [TechToThePeople/civisualize#83]
Hello Xavier,
Don't know what causes this, but is it awkward :-)

> Civisualize. The missing data visualization extension (eu.tttp.civisualize) version 1.5 is installed. Upgrade to version 1.42.

**Comment by sleewok** [comment]
I have the same issue. CiviCRM extension page is not picking up version 1.5. I had to clone the repo to get 1.5

**Comment by Upperholme** [comment]
Bump. Just installed the release labelled as "v1.5" (as opposed to the earlier release labelled "1.5") and the Extensions page at /civicrm/admin/extensions?reset=1 reports that "Version 1.42 is available. Upgrade". Judging the release dates v1.42 is nearly a year older than v1.5. Maybe it just needs renaming as v1.4.2 ?

**Comment by joemcl** [comment]
Upgraded to 4.7.17 and getting the same message - "Civisualize. The missing data visualization extension (eu.tttp.civisualize) version 1.5 is installed. Upgrade to version 1.42." even though I git cloned v1.5 .

**Comment by xurizaemon** [comment]
Ran into this also. Unfortunately it looks like a misread of how [`version_compare()`](http://php.net/manual/en/function.version-compare.php) works - assuming CiviCRM uses `version_compare()` to decide if upgrade is required, which is likely unless we rolled our own way of doing it (I didn't check).

At https://github.com/TechToThePeople/civisualize/releases, version 1.42 is a greater version than 1.5, so the next release needs to be 1.43 or greater. Many sites will now have cached that v1.42 shipped, and will do weird things until they uncache this or see a new version. Since version numbers are free and user confusion is not, I propose we just bump to v1.43 or greater and be done with it.

(Curiously, I was prompted to install v1.47. I have *no* idea how that happened.)
